README.md

```diff
@@ -84,7 +77,7 @@ Currently, tests are disabled, as the issues need to be reviewed first.
 There is also a way of running this check from command line.
 
 ```shell
-mvn test -Dtest=SecretsRegexTest#shouldValidateSingleFile -DargLine="-Dfilename=google-oauth2.yaml"
+./gradlew clean test --console plain -Dorg.gradle.caching=false --tests SecretsRegexTest.shouldValidateSingleFile -Dfilename=google-oauth2.yaml
```

Comment:
```suggestion
./gradlew test --console plain -Dorg.gradle.caching=false --tests SecretsRegexTest.shouldValidateSingleFile -Dfilename=google-oauth2.yaml
```
AFAIU, most of the time `clean` is not needed in Gradle, unlike Maven

Reply:
I know. Is mostly for AppSec to run simple test in their workflow. Maybe ` --rerun-tasks` will be better as `clean` here?

Reply:
Aah, I see. Yes, I'll add `--rerun-tasks` for now, but ideally it should be solved by marking yaml files as task inputs so that Gradle executes tasks if files have changed